### PR TITLE
Create CHANGELOG.md

### PR DESCRIPTION
**Pull request by francislavoie**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) 
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
```
Comment (xelan) [inline CHANGELOG.md]: Maybe we should order the changes

1. BC breaks
2. Security
3. New features
4. Everything else

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Change code style to PSR-2 (PR #25)
+- Update required Symfony components to 2.7+ or 3.x
+- Update required Predis version to 1.1.x
+- Fix closures with whitespace in their declaration (Issue #30)
+- Fix job stability by reconnecting to redis after forking (Commit [cadfb09e](https://github.com/mjphaynes/php-resque/commit/cadfb09e81152cf902ef7f20e6883d29e6d1373b))
+- Fix crash if the status is not set (Commit [cadfb09e](https://github.com/mjphaynes/php-resque/commit/cadfb09e81152cf902ef7f20e6883d29e6d1373b))
+- Add option to allow phpiredis support (Commit [4e22e0fb](https://github.com/mjphaynes/php-resque/commit/4e22e0fb31d8658c2a1ef73a5a44c927fd88d55c))
+- Change worker wait log level from DEBUG from INFO (Commit [4915d51](https://github.com/mjphaynes/php-resque/commit/4915d51ca2593a743cecbab9597ad6a1314bdbed))
```
Comment (xelan) [inline CHANGELOG.md]: Shouldn't this mean "from INFO to DEBUG"?

**Comment by francislavoie** [comment]
@xelan as per my comment https://github.com/mjphaynes/php-resque/pull/39#issuecomment-281044762 😜

**Comment by xelan** [comment]
Just a few comments... What do you think?

**Comment by francislavoie** [comment]
Sure, I was just ordering them chronologically. Feel free to change whatever. See here, http://keepachangelog.com/en/0.3.0/ they mention:

> Each version should:
> - List its release date in the above format.
> - Group changes to describe their impact on the project, as follows:
> - `Added` for new features.
> - `Changed` for changes in existing functionality.
> - `Deprecated` for once-stable features removed in upcoming releases.
> - `Removed` for deprecated features removed in this release.
> - `Fixed` for any bug fixes.
> - `Security` to invite users to upgrade in case of vulnerabilities.

So you can group those if you like

**Comment by xelan** [comment]
Merged, thank you :smile:

**Comment by francislavoie** [comment]
@xelan you accidentally removed a space on the DEBUG to INFO line

**Comment by xelan** [comment]
Good catch, thanks! I'll change it when merging #27, as I need to update the changelog anyway.

**Pull request merged**